src/tools/wasm-strip.cc

```diff
@@ -76,9 +88,22 @@ class BinaryReaderStrip : public BinaryReaderNop {
     return stream_.WriteToFile(filename);
   }
 
+  Result BeginCustomSection(Index section_index,
+                            Offset size,
+                            std::string_view section_name) override {
+    if (sections_to_keep_.count(section_name) > 0) {
```

Comment:
Isn't it ridiculous that we can't use `set::contains` until C++20?   I guess there must have been rationale at the time, but can you imagine designed a set API without a "contains/has" operation?

Reply:
Ikr? I was really surprised by this as well